javascript/javascript1/week4/Homework/square_every_digit.js

```diff
@@ -0,0 +1,16 @@
+console.log("===========Square Every Digit===========");
+
+//This works in VS Code, not being accepted by Codewars :(
+
+function squareDigits(num) {
+  const digits = num.toString().split("");
+  const realDigits = digits.map(Number);
+  let arrayDigits = [];
+  for (let i = 0; i < realDigits.length; i++) {
+    let squaredDigits = realDigits[i] * realDigits[i];
```

Comment:
This implementation is correct. You can also use [Math.pow()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow).